node-api/engines/echo/validator.go

```diff
@@ -153,15 +153,15 @@ func ValidateRoot(value string) bool {
 func ValidateValidatorStatus(fl validator.FieldLevel) bool {
 	// Eth Beacon Node API specs: https://hackmd.io/ofFJ5gOmQpu1jjHilHbdQQ
 	allowedStatuses := map[string]bool{
-		"pending_initialized": true,
-		"pending_queued":      true,
-		"active_ongoing":      true,
-		"active_exiting":      true,
-		"active_slashed":      true,
-		"exited_unslashed":    true,
-		"exited_slashed":      true,
-		"withdrawal_possible": true,
-		"withdrawal_done":     true,
+		utils.ActiveExiting:      true,
```

Comment:
nit: is utils the right package to list validators statuses?

Reply:
let's use the one from primitives/constants/validator.go here